doc/code/qml_qchem.rst

```diff
@@ -13,7 +13,7 @@ number observables.
 .. automodapi:: pennylane.qchem
     :no-heading:
     :include-all-objects:
-    :skip: taper, symmetry_generators, paulix_ops, import_operator
+    :skip: taper, symmetry_generators, paulix_ops, import_operator, import_state
```

Comment:
Why was this needed? Just to be consistent with `import_operator?`

Reply:
Yes -- otherwise it shows up as a function when you go to qml.qchem in the API. And `import_operator` does not, so I went  for consistency -- I think utility functions are not supposed to show up there.

Reply:
👍🏼